cmd/create/ocmrole/cmd.go

```diff
@@ -267,6 +267,10 @@ func run(cmd *cobra.Command, argv []string) {
 			})
 			os.Exit(1)
 		}
+		_, _, err = checkRoleExists(reporter, awsClient, roleNameRequested, isAdmin, aws.ModeManual)
+		if err != nil {
+			reporter.Warnf("There was an error while checking if the ocm role already exists: %s", err)
```

Comment:
reporter.Warnf(("Creating ocm role '%s' should should fail : %s", rolename, err) same as dry run option

Reply:
Changed, thanks.